src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="container">
      <div class="stars stars--0">
```
Don't leave newlines between parent and child elements
![image](https://user-images.githubusercontent.com/96409563/219939151-00074893-a019-4924-8ba4-22f79a4ad29f.png)